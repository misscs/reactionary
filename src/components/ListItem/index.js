import React from 'react'

function ListItem(props) {
  return (
    <li>
      {props.item}
    </li>
  )
}

ListItem.propTypes = {
  item: React.PropTypes.any,
}

export default ListItem
