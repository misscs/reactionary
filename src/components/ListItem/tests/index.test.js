import React from 'react'
import renderer from 'react-test-renderer'

import ListItem from 'components/ListItem'

describe('<ListItem />', () => {
  it('should render the content passed to it', () => {
    const content = (<div>Hello world!</div>)
    const component = renderer.create(
      <ListItem item={content} />
    )

    const tree=component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
