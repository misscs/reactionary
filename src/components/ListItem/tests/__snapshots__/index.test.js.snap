// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> should render the content passed to it 1`] = `
<li>
  <div>
    Hello world!
  </div>
</li>
`;
